students/lee_deitesfeld/series.py

```diff
@@ -0,0 +1,80 @@
+#--------------------------
+#Title: Number sequences
+#Description: Determine value of nth value in sequence
+#Original Dev: Lee Deitesfeld
+#Change Log:
+#20190331LAD Created functions
+#--------------------------
+
+def fibonacci(n):
+    '''Determines the nth value of the Fibonacci sequence.'''
+    #initial values of the sequence
+    a,b = 0,1
+    fib_list = []
+    #appends value to list as determined by Fibonacci calculation
+    while b < 10000000000000000000000:
+        a, b = b, a+b
+        fib_list.append(a)
+    #print(fib_list)
+
+    #presets value with index zero, else calculates value
+    if n == 0:
+        print('0')
+    else:
+        print(fib_list[n-1])
```

Comment:
your function is supposed to return nth value - right now your functions don't return any value.

also, you don't want to continue looping once you found the nth value.